src/SFA.DAS.Provider.PR.Web/Views/AddEmployer/ShutterPages/PayeAornMatchedEmailNotLinkedHasRelationshipShutterPage.cshtml

```diff
@@ -16,7 +16,7 @@
     <div class="govuk-grid-column-two-thirds">
         <p class="govuk-body">
             <div class="govuk-inset-text">
-                @Model.EmployerName <br /><br />
+                <p class="govuk-!-font-size-24">@Model.EmployerName </p>
```

Comment:
I believe wherever employer name appears it needs to be in all caps. The example given in the ticket is in all caps but I don't see here or in the model where it's set to all caps. Worth double checking with Olivia if it does need to be all caps

Reply:
in this case, the viewmodel casts it to upper

Reply:
here: https://github.com/SkillsFundingAgency/das-provider-pr-web/blob/2323c03d8629d8f597fdc29b3ba128e85133e5e5/src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/SearchByPayeController.cs#L249